syntax = "proto3";

package grpc_test_server;

service gRPCTestService{
  rpc TestServe(gRPCTestMessage) returns (gRPCTestMessage);
}

message gRPCTestMessage {
  string text = 1;
}
